packages/@aws-cdk/aws-glue/lib/table.ts

```diff
@@ -102,7 +122,16 @@ export interface TableProps {
    *
    * @default table is not partitioned
    */
-  readonly partitionKeys?: Column[]
+  readonly partitionKeys?: Column[];
+
+  /**
+   * Partition indecies on the table. A maximum of 3 indexes
+   * are allowed on a table. Keys in the index must be part
+   * of the table's partition keys.
+   *
+   * @default table has no partition indecies
```

Comment:
Either use indices or indexes (but not "indecies"). :)

```suggestion
   * @default table has no partition indexes
```